### ISSUE TITLE
 failed to add React Native 0.49.3

### ISSUE THREAD

**Issue by badrshs** [infinitered/ignite#1186]
Hi , 
I can't install ignite on my computer and i have tried some solutions but nothing worked for me 
I have uninstall react native and re install again but I got the same issue 

I have changed the name of the project to abcd but same result 
any help please ? 

```
C:\WINDOWS\system32>y:

Y:\> ignite new abcd
-----------------------------------------------
  (                  )   (
  )\ )   (        ( /(   )\ )    *   )
 (()/(   )\ )     )\()) (()/(  ` )  /(   (
  /(_)) (()/(    ((_)\   /(_))  ( )(_))  )\
 (_))    /(_))_   _((_) (_))   (_(_())  ((_)
 |_ _|  (_)) __| | \| | |_ _|  |_   _|  | __|
  | |     | (_ | | .` |  | |     | |    | _|
 |___|     \___| |_|\_| |___|    |_|    |___|
-----------------------------------------------

An unfair headstart for your React Native apps.
https://infinite.red/ignite

-----------------------------------------------

🔥 igniting app abcd
√ using the Infinite Red boilerplate v2 (code name 'Andross')
× failed to add React Native 0.49.3

Y:\> ignite doctor
System
  platform           win32
  arch               x64
  cpu                4 cores      Intel(R) Core(TM) i7-7500U CPU @ 2.70GHz
  directory          Y:\

JavaScript
  node               7.6.0        C:\Program Files\nodejs\node.EXE
  npm                4.1.2        C:\Program Files\nodejs\npm.CMD
  yarn               1.3.2        C:\Program Files (x86)\Yarn\bin\yarn.CMD

React Native
  react-native-cli   2.0.1

Ignite
  ignite             2.0.0        C:\Users\bader\AppData\Roaming\npm\ignite.CMD

Android
  java               -            C:\ProgramData\Oracle\Java\javapath\java.EXE
  android home       -            C:\Users\bader\AppData\Local\Android\sdk

Y:\>
```

**Comment by GantMan** [comment]
That's very very odd.  And you say you can create a new react native project just fine?

@kevinvangelder - this is windows, can you join in and help here?

**Comment by badrshs** [comment]
Hi ,  i had removed  yarn package and now react native installation work perfecat without any error 
but unfortunately to install ignite I must install yarn  😢

**Comment by kevinvangelder** [comment]
If memory serves, Yarn is preferred but not required, so feel free to give it a shot without Yarn. From what you've posted above it does indeed appear to be an issue between `react-native init` and `yarn` and the trace doesn't really give much clue as to why. Looks like some other people have had a similar issue, maybe give their solution a shot? https://github.com/yarnpkg/yarn/issues/3227

**Comment by badrshs** [comment]
thank you 
then , how can i install ignite without yarn ? ? 
```

Y:\> ignite new PizzaApp
  Error: not found: yarn

  - which.js:13 getNotFoundError
    [npm]/[ignite-cli]/[which]/which.js:13:12

  - which.js:131 Function.whichSync [as sync]
    [npm]/[ignite-cli]/[which]/which.js:131:9

  - system-extension.js:113 Object.which
    [npm]/[ignite-cli]/[gluegun]/src/core-extensions/system-extension.js:113:22

  - ignite.js:41 Object.attach [as setup]
    [npm]/[ignite-cli]/src/extensions/ignite.js:41:39

  - runtime.js:147 extension
    [npm]/[ignite-cli]/[gluegun]/src/domain/runtime.js:147:34

  - forEach.js:43 forEach
    [npm]/[ignite-cli]/[ramda]/src/forEach.js:43:5

  - _checkForMethod.js:22
    [npm]/[ignite-cli]/[ramda]/src/internal/_checkForMethod.js:22:10

  - _curry2.js:25 f2
    [npm]/[ignite-cli]/[ramda]/src/internal/_curry2.js:25:16

  - runtime.js:145 Runtime.run
    [npm]/[ignite-cli]/[gluegun]/src/domain/runtime.js:145:5

  - cli.js:65 run
    [npm]/[ignite-cli]/src/cli/cli.js:65:29


(node:5248) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 2): Error: not found: yarn
(node:5248) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

Y:\>
```

**Comment by kevinvangelder** [comment]
Strange, we put a lot of effort building it in such a way that it would work with or without yarn, so I'm not sure when that changed. 😞

**Comment by jamrizzi** [comment]
It definitely does not work without yarn

**Comment by jamrizzi** [comment]
Which is a slight frustration now that npm is basically equivalent

**Comment by GantMan** [comment]
agreed.  Going to have to dig into why yarn is an issue here.  PR welcome since you have the machine without yarn, maybe it's something simple?

**Comment by jamrizzi** [comment]
I have filed this as a separate issue at #1198

**Comment by jamrizzi** [comment]
I have fixed this with #1199
